fabric-resource-loader-v0/src/main/java/net/fabricmc/fabric/impl/resource/loader/FabricModResourcePack.java

```diff
@@ -47,7 +47,7 @@ public InputSupplier<InputStream> openRoot(String... pathSegments) {
 		String fileName = String.join("/", pathSegments);
 
 		if ("pack.mcmeta".equals(fileName)) {
-			String description = "Mod resources.";
+			String description = Text.translatable("pack.fabricapi.description");
 			String pack = String.format("{\"pack\":{\"pack_format\":" + type.getPackVersion(SharedConstants.getGameVersion()) + ",\"description\":\"%s\"}}", description);
```

Comment:
```suggestion
			String description = "pack.description.fabricApi";
			String pack = String.format("{\"pack\":{\"pack_format\":" + type.getPackVersion(SharedConstants.getGameVersion()) + ",\"description\":{\"translate\":\"%s\"}}}", description);
```
also fixes the incorrect translation key